### PR TITLE
Group assignment 5

### PR DESCRIPTION
**Pull request by ileanapopa11**

### CODE REVIEW

File: group/assignment-5/src/main/java/Main.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        List<String> wordList = null;
+        try {
+            wordList = getWordList("dictionary.in");
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+
+        LetterTrie dictTrie = new LetterTrie(wordList);
+
+        Alphabet a = new Alphabet(dictTrie);
```
[the remark this answers is not in the document]
Comment (ileanapopa11) [reply group/assignment-5/src/main/java/Main.java]: You're right. This way, the worldList could potentially be obtained in different ways, not necessarily from a Trie. Thank you.

File: group/assignment-5/src/test/java/TestSolutionOne.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    public void findAlphabetABC(){
+        String[] words = {"abc", "bca"};
+        ArrayList<Character> alphabet = new ArrayList<>(Arrays.asList('a','b','c'));
+        ArrayList<Character> result = o.findAlphabet(words);
+
+        Assert.assertEquals(alphabet,result);
+    }
+
+
+}
```
Comment (ileanapopa11) [inline group/assignment-5/src/test/java/TestSolutionOne.java]: Nice! I think an interesting idea, if we have time, would be to test against a real dictionary. I found one in json format here: https://github.com/matthewreagan/WebstersEnglishDictionary

Comment (ileanapopa11) [reply group/assignment-5/src/test/java/TestSolutionOne.java]: I've tried it and it also works, but how about writing a test for a dictionary with one word and one for an empty dictionary?
